cli.js

```diff
@@ -4,28 +4,35 @@ const fs = require('fs');
 const meow = require('meow');
 const prettyBytes = require('pretty-bytes');
 const gzipSize = require('gzip-size');
+const chalk = require('chalk');
 
 const cli = meow(`
 	Usage
 	  $ gzip-size <file>
 	  $ cat <file> | gzip-size
 
 	Options
-	  --level  Compression level [0-9] (Default: 9)
-	  --raw    Display value in bytes
+	  --level             Compression level [0-9] (Default: 9)
+	  --raw               Display value in bytes
+	  --include-original  Display original size
 
 	Examples
 	  $ gzip-size unicorn.png
 	  192 kB
 	  $ gzip-size unicorn.png --raw
 	  192256
+	  $ gzip-size unicorn.png --include-original
+	  20 kB → 2 kB
```

Comment:
For consistency, the gzip size should be the same as in the other examples.

Reply:
`1920 kB → 192 kB`?